src/components/settings/Settings.h

```diff
@@ -98,7 +128,7 @@ namespace Pinetime {
         uint32_t stepsGoal = 1000;
         uint32_t screenTimeOut = 15000;
 
-        WakeUpMode wakeUpMode = WakeUpMode::None;
+        uint8_t wakeUpMode = static_cast<uint8_t>(WakeUpMode::None);
```

Comment:
Maybe we can investigate the use of std::bitset for these flags? Bitsets provides more readable API (set(), clear(),...) and might allow to write the code without all these static_cast.

I haven't used bitset that much, so I might be wrong...

Reply:
I've played around a bit with bitsets, it would still require the static casts.

Reply:
This new push should convert all of those to bitsets